run-dljc.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
If you use bashisms, then using bash is good.  Otherwise, it's often better to stick to sh for portability and speed.